benchmarks/nemo/nemolite2d/manual_versions/psykal_opencl/Makefile

```diff
@@ -0,0 +1,36 @@
+# Makefile for manual OpenCL version of PSyKAl'd NEMOLite2D.
+#
+# This Makefile picks up the compiler to use any flags from
+# environment variables.
+
+LDFLAGS += ${OMPFLAGS}
+FFLAGS += ${OPENCL_INCLUDE}
+
+.PHONY: all nemolite2d.exe device_binary
+.DEFAULT_GOAL := nemolite2d.exe
+include ../common.mk
+
+COMMON_MODULES = time_step_mod.o nemolite2d.o
+
+all: nemolite2d.exe
+
+nemolite2d.exe: ${COMMON_LIB} inf_lib timer_lib fcl_lib ${COMMON_MODULES}
+	${F90} -o $@ ${COMMON_MODULES} ${OPENCL_LIBS} ${COMMON_LIB} \
+        ${FCL_LIB} ${INF_LIB} ${TIMER_LIB} -lOpenCL $(LDFLAGS)
+
+device_binary: %.cl
```

Comment:
I added a comment explaining its purpose and how to use it. However, this target is the one I need to play a bit more (after this PR) in order to be confident it works across different target accelerators/compilers. So maybe it will be polished in the near future.